packages/compass-e2e-tests/helpers/commands/save-connection-string-as-favorite.ts

```diff
@@ -1,16 +1,21 @@
 import { UUID } from 'bson';
 import type { CompassBrowser } from '../compass-browser';
 import * as Selectors from '../selectors';
+import { TEST_MULTIPLE_CONNECTIONS } from '../compass';
 
 export async function saveConnectionStringAsFavorite(
   browser: CompassBrowser,
   connectionString: string,
   favoriteName?: string,
-  color: `color${number}` = 'color1'
+  color: `color${number}` | string = 'color1'
 ): Promise<string> {
+  if (TEST_MULTIPLE_CONNECTIONS && color === 'color1') {
```

Comment:
This is a total hack for now, but this whole command should just go away when we remove single connections.